Linode power updates

eatonphil opened the pull request:
* Add delete to Linode action dropdown, closes #582 
* Linode power actions should have confirm, closes #1940, closes #1606 
  * All but power on, which shouldn't need a confirm.
* Allow you to launch lish no matter the power state, closes #1939 
* Rebuild triggers statusdropdown polling, closes #1986
* Polls on mass events, closes #1740
* Updates Polling object to allow one request for all polling ids, closes #1452 
* Adds default first button action that opens dropdown, closes #1947
* Changes behavior enough relevant to #1989 such that I cannot reproduce it, closes #1989

na3d commented:
Linter Error
> src/linodes/components/StatusDropdown.js
> 50:5  warning  Unexpected console statement  no-console

mparke commented on src/api/polling.js:
```diff
@@ -65,9 +68,18 @@ export default function Polling(args) {
     }
   }
 
+  function isPolling(id) {
```
nice

mparke commented:
reboot confirm doesn't trigger status animation?

eatonphil commented:
I moved the progress logic out of the API coded and implemented a separate progress bar so you can still see what the Linode is doing while it's transitioning.
<img width="1138" alt="screen shot 2017-06-01 at 8 06 59 pm" src="https://cloud.githubusercontent.com/assets/3925912/26705750/2d681038-4706-11e7-8df8-35fd54f68623.png">

eatonphil commented:
Fixes selected Linode background color too. We probably want a different color though...
<img width="1146" alt="screen shot 2017-06-01 at 8 21 20 pm" src="https://cloud.githubusercontent.com/assets/3925912/26706031/2cb90a82-4708-11e7-9333-8778c6ad583b.png">

mparke commented:
For later: one thing we may want to try and improve further is the rebuild status change, it goes from powering down > offline ( which is cool ), but then nothing else until the events notification comes up that it was rebuilt.
